next-app/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "next-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
```

Comment:
update some of these to run bazel instead?

Reply:
👍

Reply:
however, it doesn't work with notify changes